include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -202,7 +202,20 @@ typedef enum OrtMemType {
   OrtMemTypeDefault = 0,                // the default allocator for execution provider
 } OrtMemType;
 
+struct OrtApi;
+typedef struct OrtApi OrtApi;
+
+struct OrtApiBase {
+  const OrtApi*(ORT_API_CALL* GetApi)(uint32_t version)NO_EXCEPTION;  // Pass in ORT_API_VERSION
+  const char*(ORT_API_CALL* GetVersionString)() NO_EXCEPTION;
+};
+typedef struct OrtApiBase OrtApiBase;
+
+ORT_EXPORT const OrtApiBase* ORT_API_CALL OrtGetApiBase() NO_EXCEPTION;  // Pass in ORT_API_VERSION
```

Comment:
Do we need this comment "// Pass in ORT_API_VERSION" on this line?

Reply:
Nope, good catch